NPointFunctions code generation bug if no FormCalc is installed

### Bug description

Disabling `FormCalc` breaks code generation, resulting in
```
models/SM/observables/SM_br_l_to_3l.cpp:26:118: warning: extra tokens at end of #include directive
   26 | #include <valarray>                                                                                                  const std::array<int,4>&,
      |                                                                                                                      ^~~~~
models/SM/observables/SM_br_l_to_3l.cpp:27:118: warning: extra tokens at end of #include directive
   27 | #include <complex>
....
```
### Steps to reproduce

```
./configure --with-models=SM --disable-formcalc
make
```

--- a/meta/FlexibleSUSY.m
+++ b/meta/FlexibleSUSY.m
@@ -4228,8 +4228,8 @@ SetupOutputParameters[massMatrices_] :=
 
 CheckObsDependencies[requested_List] :=
 Module[{allObs, dir, filtered = requested},
+   Needs@"NPointFunctions`";
    If[FlexibleSUSY`FSFeynArtsAvailable && FlexibleSUSY`FSFormCalcAvailable,
-      Needs@"NPointFunctions`";
       Needs@"WilsonCoeffs`";
       Return[filtered];
    ];

